### PR TITLE
Add unit tests for class feign.Util

### PR DESCRIPTION
**Pull request by petersear**
Hi,
I've analysed your codebase and added some unit tests for the class feign.Util.
I've written the tests for these functions with the help of [Diffblue](https://www.diffblue.com/) [Cover](https://static1.squarespace.com/static/5bc9dd6fb2cf795fa3511f70/t/5be2e84f8985836d62e8eb13/1541597264010/Diffblue+Cover+Datasheet.pdf).
Hopefully, these tests should help you detect regressions caused by future code changes.

### CODE REVIEW

**Comment by paulkennethkent** [comment]
Make sure the link "Diffblue Cover" links to the website.

File: core/src/test/java/feign/UtilTest.java [hunk 1]
```diff
@@ -121,6 +124,145 @@ public void unboundWildcardIsObject() throws Exception {
...
+    final String value = "AAAAAAAA";
+
+    // Act
+    final boolean retval = Util.isNotBlank(value);
+
+    // Assert result
+    Assert.assertEquals(true, retval);
+  }
 
   interface LastTypeParameter {
```
Comment (paulkennethkent) [inline core/src/test/java/feign/UtilTest.java:239]: I'm slightly confused what type of file you are adding these tests to?

Comment (petersear) [reply core/src/test/java/feign/UtilTest.java:239]: This file contains the existing tests for the class core.src.test.java.feign.UtilTest.
As they seemed to be in no particular order, I added our tests together in a block after all the others.

**Comment by petersear** [comment]
Good catch re. the links @paulkennethkent , I'll update the Markdown in the "Submit PR..." doc.

**Review by JohnLBergqvist — APPROVED** [review]

**Review by sudo-jaa — APPROVED** [review]

**Pull request closed without merging**